docs/protocol/README.md

```diff
@@ -1,3 +1,896 @@
 # Wesh protocol
 
 The Wesh protocol is documented at https://berty.tech/docs/protocol .
+## Introduction
```

Comment:
Can you rebase to the latest commit in main? This file was changed in a previous commit.

Reply:
looks in already on top of the main branch

Reply:
I don't understand why it is changed. Here is the latest in weshnet main:
https://github.com/berty/weshnet/blob/main/docs/protocol/README.md
And here it is in your branch:
https://github.com/gfanton/weshnet/blob/fix/go-mod/docs/protocol/README.md

Reply:
(There may be other files from an old snapshot.)